roles/module_openapi/tasks/main.yaml

```diff
@@ -11,7 +11,7 @@
   register: tmp_directory
 
 - name: EXECUTE the python script
-  ansible.builtin.command: python3 doc_generator.py "{{ plugin['rm_swagger_json'] }}"
+  ansible.builtin.command: python3 "{{ role_path }}"/templates/doc_generator.py "{{ plugin['rm_swagger_json'] }}"
```

Comment:
Keeping doc_generator as a Python script makes this task flaky for a variety of reasons, one of them being the invocation of python with `python3`, overriding the value of `ansible_python_interprerter`, which is the one expected to be used.

For long term stability, it is better to convert this into an plugin (preferably filter) like some of the already existing ones in this collection, and let ansible-core's plugin resolution algorithm take care of loading it as well as executing it with the correct interpreter.

Reply:
Yea, plan is to change the doc_generator logic to plugin, currently this fix is address the issue that was raised against the functioning of the sec builder logic.

Reply:
@NilashishC I've added #38, for converting doc_generating script to plugin logic